Dockerfile

```diff
@@ -26,4 +33,13 @@ RUN yarn global add node-gyp@6.1.0 && \
     yarn build && \
     yarn cache clean
 
+# Run container as a non-root user
+RUN adduser \
+    --disabled-password \
+    --home /usr/src/app \
+    --gecos '' app \
+    && chown -R app /usr/src/app
+
+USER app
+
```

Comment:
Node images already provide a `node` user, remove the `adduser` command and instead replace with:

```Dockerfile
USER node
```